mantis-control-plane/mantis-control-plane-core/build.gradle

```diff
@@ -32,17 +32,28 @@ dependencies {
     api project(":mantis-common")
 
     api "org.skife.config:config-magic:$configMagicVersion"
+    api 'com.google.code.findbugs:jsr305:3.0.1' // For Nonnull annotation
     api libraries.flinkRpcApi
-    api libraries.flinkRpcImpl
     api libraries.flinkCore
+    compileOnly libraries.flinkRpcImpl // Provided by copyLibs task
 
     api "org.apache.mesos:mesos:$mesosVersion"
     api "org.json:json:$jsonVersion"
     api "org.hdrhistogram:HdrHistogram:$hdrHistogramVersion"
     api "com.github.spullara.cli-parser:cli-parser:$cliParserVersion"
     api "joda-time:joda-time:$jodaTimeVersion"
 
-
+    testFixturesImplementation "org.apache.flink:flink-rpc-akka:1.14.2"
     testImplementation libraries.junit4
     testImplementation libraries.mockitoAll
 }
+
+task copyLibs(type: Copy) {
```

Comment:
Could you add a comment here about what this is doing?

Reply:
+1